server/firebase.js

```diff
@@ -31,18 +31,8 @@ firebaseRequests.read = async function(table, uid) {
 };
 
 firebaseRequests.verifyIdToken = async function(idToken) {
-  return await auth.verifyIdToken(idToken);
-    // .then(function(decodedToken) {
-    //   console.log("Decoded token: ", decodedToken);
-    //   return true;
-    //   // var uid = decodedToken.uid;
-    //   // ...
-    // }).catch(function(error) {
-    //   // Handle error
-    //   // TODO
-    //   console.log("Invalid token\n")
-    //   return false;
-    // });
-}
+    var tokenData = await auth.verifyIdToken(idToken);
+    return tokenData;
```

Comment:
is breaking this up just stylistic or is there something i should know about combining return statements & awaits?

Reply:
nah functionally i don't think it's any different. just makes it a little more clear what's coming back. verifyIdToken kinda sounds like it could just return a boolean. tokenData at least makes it more obvious that there's some sort of data in there. could just have a comment though too.

Reply:
just fyi the await makes it so that we don't need to use .then(...)

basically whatever code you'd do in the then function, can just do that after the call. i personally find it easier to read and write this way, but don't have to use it